src/catkin_pkg/package_templates.py

```diff
@@ -41,6 +41,8 @@
 from catkin_pkg.package import Package
 from catkin_pkg.package import PACKAGE_MANIFEST_FILENAME
 from catkin_pkg.package import Person
+from catkin_pkg.cmake import get_metapackage_cmake_template_path
+from catkin_pkg.cmake import configure_file
```

Comment:
Both lines are now in the correct order but they still need to be moved up by 4 lines to be above the `from catkin_pkg.package` lines.